jmh-core/src/test/java/org/openjdk/jmh/util/TestUtil.java

```diff
@@ -27,13 +27,24 @@
 import junit.framework.Assert;
 import org.junit.Test;
 
+import java.io.IOException;
 import java.util.Arrays;
 
 public class TestUtil {
 
     @Test
-    public void testPID() {
-        Assert.assertFalse(Utils.getPid() == 0);
+    public void testPID_Current() {
+        Assert.assertNotSame(0, Utils.getPid());
```

Comment:
This should be an assertNotEquals no? Because assertNotSame will box the 0 and will probably always be not same no? Or an hardcoded 0 is always the same?

Reply:
Right. Back to `assertTrue`, as current JUnit version does not have `assertThat`, and there is no `assertNotEquals`.